tutorials/advanced_features/Backends for Sequence Execution.ipynb

```diff
@@ -526,20 +531,20 @@
    "id": "b7daaec2",
    "metadata": {},
    "source": [
-    "We can see that there is only one `Results` object in the list after the job is done since `runs=1`. Note that since we ran the same sequence, as locally, the fidelity will be the same as in that case."
+    "We can see that there is only one `Results` object in the list after the job is done since only a single item was present in `job_params` above. Note that since we ran the same sequence as locally, the fidelity will is the same as in that case."
```

Comment:
This time we specify only one evaluation_time, do we maybe want to replicate the local example exactly?
Also 
```suggestion
    "We can see that there is only one `Results` object in the list after the job is done since only a single item was present in `job_params` above. Note that since we ran the same sequence as locally, the fidelity is the same as in that case."
```